CASSANDRA-15503 3.11 Fix CQL formatting of read commands

adelapena opened the pull request.

adelapena committed:
Fix CQL formatting of read command restrictions
patch by Andres de la Peña; reviewed by Berenguer Blasi for CASSANDRA-15503

bereng commented:
@adelapena, wondering if `DataRange#getOperator(boolean, boolean)` would need a similar treatment?

adelapena commented:
@bereng you are right, there are some errors with how token inequalities are printed, I've fixed the arguments that are passed of `getOperator` when token ranges are used, and added a few additional dtests ([here](https://github.com/apache/cassandra-dtest/pull/67/commits/02b24c1851f9dc6912313dfae14a8aef24dab1a1)). Good catch.
